modules/nvidia_plugin/src/cuda_compiled_model.cpp

```diff
@@ -0,0 +1,386 @@
+        auto model_name = model_->get_friendly_name();
+        return decltype(ov::model_name)::value_type{model_name};
+    } else if (ov::optimal_number_of_infer_requests == name) {
+        const unsigned value = memory_pool_->Size();
+        return decltype(ov::optimal_number_of_infer_requests)::value_type{value};
+    } else if (ov::execution_devices == name) {
+        return decltype(ov::execution_devices)::value_type{get_plugin()->get_device_name() + "." + std::to_string(config_.get_device_id())};
+    } else if (ov::loaded_from_cache == name) {
+        return decltype(ov::loaded_from_cache)::value_type{loaded_from_cache_};
+    } else {
+        return config_.get(name);
+    }
+}
+
+std::shared_ptr<const ov::Model> CompiledModel::get_runtime_model() const {
+    auto model = model_->clone();
+    // Add execution information into the model
+    size_t exec_order = 0;
+    for (const auto& op : model->get_ordered_ops()) {
+        auto& info = op->get_rt_info();
+        const auto& it = info.find(ov::nvidia_gpu::PERF_COUNTER_NAME);
+        OPENVINO_ASSERT(it != info.end(), "Operation ", op, " doesn't contain performance counter");
+        auto perf_count = it->second.as<std::shared_ptr<ov::nvidia_gpu::PerfCounts>>();
+        OPENVINO_ASSERT(perf_count, "Performance counter is empty");
+        info[ov::exec_model_info::LAYER_TYPE] = op->get_type_info().name;
+        info[ov::exec_model_info::EXECUTION_ORDER] = std::to_string(exec_order++);
+        info[ov::exec_model_info::IMPL_TYPE] = perf_count->impl_type;
+        auto perf_count_enabled = config_.get(ov::enable_profiling.name()).as<bool>();
+        info[ov::exec_model_info::PERF_COUNTER] = perf_count_enabled && perf_count->average() != 0
+                                                      ? std::to_string(perf_count->average())
+                                                      : "not_executed";
+
+        std::string original_names = ov::getFusedNames(op);
+        if (original_names.empty()) {
+            original_names = op->get_friendly_name();
+        } else if (original_names.find(op->get_friendly_name()) == std::string::npos) {
+            original_names = op->get_friendly_name() + "," + original_names;
+        }
+        info[ov::exec_model_info::ORIGINAL_NAMES] = original_names;
+        info[ov::exec_model_info::RUNTIME_PRECISION] = perf_count->runtime_precision;
+
+        std::stringstream precisions_ss;
+        for (size_t i = 0 ; i < op->get_output_size(); i++) {
+            if (i > 0) precisions_ss << ",";
+            precisions_ss << op->get_output_element_type(i);
+        }
+        info[ov::exec_model_info::OUTPUT_PRECISIONS] = precisions_ss.str();
+    }
+    return model;
+}
+
+void CompiledModel::export_model(std::ostream& model_stream) const {
+    OV_ITT_SCOPED_TASK(itt::domains::nvidia_gpu, "CompiledModel::export_model");
+
+    std::stringstream xml_file, bin_file;
+    auto& rt_info = model_->get_rt_info();
```

Comment:
You can use `has_rt_info()` and `get_rt_info<int64_t>()`

Reply:
Is it needed at all? I think old IRs can be exported as IRv11

Reply:
Replaced using `has_rt_info()` and `get_rt_info<int64_t>()`